fxa-auth-db-server/index.js

```diff
@@ -117,6 +121,18 @@ function createServer(db) {
 
   api.get('/__heartbeat__', reply(db.ping))
 
+  api.del(
+    '/account/:uid/device/:deviceId',
+    function (req, res, next) {
+      db.deleteDevice(req.params.uid, parseInt(req.params.deviceId, 10) || 0)
+        .then(
+          handleSuccess.bind(null, req, res),
+          handleError.bind(null, req, res)
+        )
+        .done(next, next)
+    }
+  )
```

Comment:
There's an asymmetry here, updating a device record via `POST /device` but deleting it via `DELETE /device/:deviceId`.  For other datatypes we've been relying on fxa-auth-server to generate the ids and pass them through to us in the URL, should we do the same with devices for consistency?